src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -61,23 +61,16 @@ class Crawler implements \Countable, \IteratorAggregate
     private $html5Parser;
 
     /**
-     * @param mixed     $node           A Node to use as the base for the crawling
-     * @param string    $uri            The current URI
-     * @param string    $baseHref       The base href value
-     * @param bool|null $useHtml5Parser Whether the Crawler should use the HTML5 parser or the native DOM parser
+     * @param mixed  $node        A Node to use as the base for the crawling
+     * @param string $uri         The current URI
+     * @param string $baseHref    The base href value
+     * @param HTML5  $html5Parser A default HTML5 parser instance
      */
-    public function __construct($node = null, string $uri = null, string $baseHref = null, bool $useHtml5Parser = null)
+    public function __construct($node = null, string $uri = null, string $baseHref = null, HTML5 $html5Parser = null)
```

Comment:
Old code passing a boolean as fourth parameter will break after this change.

Reply:
This PR updates code introduced a few days ago, no BC break here :) .

Reply:
that's not an issue as change with this boolean flag wasnt' released yet

Reply:
I see, thanks.